com.unity.render-pipelines.high-definition/Runtime/Material/Fabric/FabricRaytracing.hlsl

```diff
@@ -39,12 +39,21 @@ void FitToStandardLit( SurfaceData surfaceData
     outStandardlit.fresnel0 = surfaceData.specularColor;
     outStandardlit.coatMask = 0.0;
     outStandardlit.emissiveAndBaked = builtinData.bakeDiffuseLighting * surfaceData.ambientOcclusion + builtinData.emissiveColor;
-#ifdef LIGHT_LAYERS
-    outStandardlit.renderingLayers = builtinData.renderingLayers;
-#endif
-#ifdef SHADOWS_SHADOWMASK
-    outStandardlit.shadowMasks = BUILTIN_DATA_SHADOW_MASK;
-#endif
+    outStandardlit.isUnlit = 0;
+}
+
+void FitToMinimalLit( SurfaceData surfaceData
+                        , BuiltinData builtinData
```

Comment:
We don't need an extra FitToMinimalLit function here, we can simply nullify what isn't required in the RaytracingGbuffer pass directly. (let's talk). And FitToMinimalLit => guess we shouldn't call this minimal fit, let's be clear on our intend and say it is for raytrace indirect diffuse.